apps/data-generator/libs/ExchangeClient.py

```diff
@@ -1,73 +1,56 @@
 from enum import Enum
 from InsertOrder import InsertOrder, LifeSpan, Side
-from LoginResponse import LoginResponse
-import socket, errno, sys, select
+import sys
 import exchange_pb2 as proto
 import State
 from time import sleep
 from typing import Callable
 from threading import Thread
 
+from websocket import WebSocketApp
+
 HEADER_LENGTH = 8
 
 class ExchangeClient:
     '''
     A client class responsible for sending InsertOrder object through socket.
     '''
     def __init__(self, hostname: str = '127.0.0.1', port: int = 15001):
-        self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        self.socket.connect((hostname, port))
-        self.socket.setblocking(True)
-
+        
+        self.uri = f'wss://{hostname}:{port}'
```

Comment:
does wss work (idk if we are gonna end up using SSL for this lol)

Reply:
Should I just keep ws for now xD